### ISSUE TITLE
[Bug] Bad css dark reader

### ISSUE THREAD

**Issue by Tnr1112** [darkreader/darkreader#9848]
# Bug Report

## Website Address
[www.youtube.com](www.youtube.com)
[www.github.com](www.github.com)

## Bug Description
When I use the dark reader extension it breaks the css of the page. It gets solved when I disable and enable the extension.

## Steps To Reproduce

- Go to github.com.
- Observe its style.

## Expected Behavior
It should look good

## Actual Behavior
It looks ugly. Look the screenshots

## Screenshots
![darkReader](https://user-images.githubusercontent.com/51060167/190873830-182d1afe-0851-4e53-afca-bb47c5ed5f55.png)
![githubBug](https://user-images.githubusercontent.com/51060167/190873831-e8338d93-b4a7-45eb-9d5e-9204a40ab52d.png)

## System Information

- OS: Windows
- Browser: Chrome 105.0.5195.126
- Dark Reader Version: 4.9.57

## Additional Context

The urls that I passed are test urls, this bug is happening to me in all the pages I access.

**Comment by Gusted** [comment]
Hi @Tnr1112!

This indeed a bug, however it's quite weird that you're facing this bug. Could you try to re-install Dark Reader? That might solve the problem. Otherwise please check if there are any errors reported in the console(ctrl + shift + i), try to do this on a small site, e.g. https://example.com/.

**Comment by Tnr1112** [comment]
Hi, I have already reinstalled, but it didn't fix it. In the [https://example.com/](https://example.com/) doesn't works too. I found when Dark Reader breaks down. If I have the same page open in two differents tabs, I disable and enable the extension to fix that problem, then the other one breaks down too. Thanks for your reply!

Example.com broken
![image](https://user-images.githubusercontent.com/51060167/191500861-ca069ea8-5ad4-46ce-abf2-56b7b5f2f1f2.png)

Example.com working properly
![image](https://user-images.githubusercontent.com/51060167/191500916-d88e881b-5c37-4098-85a1-ed14d2ff1df3.png)
